test/tla/cli-integration-tests.md

```diff
@@ -421,6 +421,16 @@ The outcome is: NoError
 ...
 ```
 
+### check APAbcastByz.tla succeeds
+
```

Comment:
@konnov perhaps this would also be a good place to document why we want to test this complicated spec specifically? Perhaps you could provide the context needed for that?

Reply:
Let's give a link to https://github.com/informalsystems/apalache/issues/157